grafana-plugin/src/pages/integration/Integration.tsx

```diff
@@ -697,6 +692,17 @@ class Integration extends React.Component<IntegrationProps, IntegrationState> {
       })
       .finally(() => this.setState({ isLoading: false }));
   }
+
+  async loadContactPointsMaybe(id: AlertReceiveChannel['id']) {
```

Comment:
let's maybe use more general name like `loadExtraData`, and inside that method we will check which data do we need depending on integration type and load it